File: C#Integration/PokerProtocol/Commands/Game/GameEndedCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using EricUtility;

namespace PokerProtocol.Commands.Game
{
    public class GameEndedCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return GameEndedCommand.COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "gameENDED";


        public GameEndedCommand(StringTokenizer argsToken)
        {
        }

        public GameEndedCommand()
        {
        }
    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands2/game/GameEndedCommand.java
package bluffinmuffin.protocol.commands2.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands2.AbstractCommand;

public class GameEndedCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return GameEndedCommand.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "gameENDED";
    
    public GameEndedCommand(StringTokenizer argsToken)
    {
    }
    
    public GameEndedCommand()
    {
    }
}
